### PR TITLE
Adds ML Director Insights to Blog

### PR DESCRIPTION
**Pull request by BritneyMuller**
Adds ML Director Insights Interviews to blog:
screencapture-localhost-5564-blog-ml-director-insights-2022-04-21-11_57_20

### CODE REVIEW

**Review by BritneyMuller — COMMENTED** [review]
Let me know if you see any fixes, this looks good to me.

File: ml-director-insights.md [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+    </a>
+</div>
+</head>
+<body>
+
+Few seats at the Machine Learning table span both technical skills and business acumen like Directors of Machine Learning.
+
+Directors of Machine Learning and/or Data Science are often expected to design ML systems, have deep knowledge of mathematics, familiarity with ML frameworks, rich data architecture understanding, experience applying ML to real-world applications, solid communication skills, and often expected to keep on top of industry developments. A tall order!
+<a href="https://huggingface.co/platform?utm_source=article&utm_medium=blog&utm_campaign=ml_director_insights_1"><img width="100%" align=
+"right" src="/blog/assets/61_ml_director_insights/private-model-hub.png"></a>
```
Comment (feconroses) [inline ml-director-insights.md:36]: Change:
```
A tall order!
<a href="https://huggingface.co/platform?utm_source=article&utm_medium=blog&utm_campaign=ml_director_insights_1"><img width="100%" align=
"right" src="/blog/assets/61_ml_director_insights/private-model-hub.png"></a>
````

to:
```suggestion
A tall order!

<a href="https://huggingface.co/platform?utm_source=article&utm_medium=blog&utm_campaign=ml_director_insights_1"><img width="100%" style=
"margin:auto" src="/blog/assets/61_ml_director_insights/private-model-hub.png"></a>
```

File: ml-director-insights.md [hunk 2]
```diff
@@ -0,0 +1,247 @@
...
+            <span class="fullname">Britney Muller</span>
+        </div>
+    </a>
+</div>
+</head>
+<body>
+
+Few seats at the Machine Learning table span both technical skills and business acumen like Directors of Machine Learning.
+
+Directors of Machine Learning and/or Data Science are often expected to design ML systems, have deep knowledge of mathematics, familiarity with ML frameworks, rich data architecture understanding, experience applying ML to real-world applications, solid communication skills, and often expected to keep on top of industry developments. A tall order!
```
Comment (feconroses) [inline ml-director-insights.md:34]:
```suggestion
They are often expected to design ML systems, have deep knowledge of mathematics, familiarity with ML frameworks, rich data architecture understanding, experience applying ML to real-world applications, solid communication skills, and often expected to keep on top of industry developments. A tall order!
```

File: ml-director-insights.md [hunk 3]
```diff
@@ -0,0 +1,247 @@
...
+<a href="https://huggingface.co/platform?utm_source=article&utm_medium=blog&utm_campaign=ml_director_insights_1"><img width="100%" align=
+"right" src="/blog/assets/61_ml_director_insights/private-model-hub.png"></a>
+
+For these reasons, we’ve tapped into this unique group of ML experts for a series of articles highlighting their thoughts on current ML insights and industry trends ranging from Healthcare to Finance, eCommerce, SaaS, Research, Media, and more. For example, one ML Director will note how ML can be used to reduce empty deadheading truck driving (which occurs ~20% of the time) down to just 19% would cut carbon emissions by ~100,000 Americans. Note: This is back of napkin math, done by an ex-rocket Scientist though, so we’ll take it.
```
Comment (feconroses) [inline ml-director-insights.md:38]:
```suggestion
For these reasons, we’ve tapped into this unique group of ML Directors for a series of articles highlighting their thoughts on current ML insights and industry trends ranging from Healthcare to Finance, eCommerce, SaaS, Research, Media, and more. For example, one ML Director will note how ML can be used to reduce empty deadheading truck driving (which occurs ~20% of the time) down to just 19% would cut carbon emissions by ~100,000 Americans. Note: This is back of napkin math, done by an ex-rocket Scientist though, so we’ll take it. 
```

**Review by feconroses — APPROVED** [review]
@BritneyMuller really enjoyed this content! Some comments:

- I'm unsure if the old commit "Adds Lewis Tunstall interview to blog" in this PR has any negative effect. Just in case, let's wait for Omar or Julien's comments or ask in 
#hub-server-support if it's ok to merge.
- Can you please optimize the image `private-model-hub.png`?
- The "Request a demo" on the CTA image is a bit small and difficult to read. Maybe you can slightly increase the font size here?

File: ml-director-insights.md [hunk 4]
```diff
@@ -0,0 +1,247 @@
...
+
+
+
+<img style="float: left;" padding="5px" src="/blog/assets/61_ml_director_insights/Archi-Mitra.jpeg"></a>  
+
+### [Archi Mitra](https://www.linkedin.com/in/archimitra/) - Director of Machine Learning at [Buzzfeed](https://www.buzzfeed.com/)
+
+**Background:** Bringing balance to the promise of ML for business. People over Process. Strategy over Hope. AI Ethics over AI Profits. Brown New Yorker.
+
+**Fun Fact:** I can speak Dzongkha (google it!) and am a supporter of [Youth for Seva](https://www.youthforseva.org/Donation).
```
Comment (osanseviero) [inline ml-director-insights.md:52]: Maybe we could add a link to Dzongkha

File: ml-director-insights.md [hunk 5]
```diff
@@ -0,0 +1,247 @@
...
+"right" src="/blog/assets/61_ml_director_insights/private-model-hub.png"></a>
+
+For these reasons, we’ve tapped into this unique group of ML experts for a series of articles highlighting their thoughts on current ML insights and industry trends ranging from Healthcare to Finance, eCommerce, SaaS, Research, Media, and more. For example, one ML Director will note how ML can be used to reduce empty deadheading truck driving (which occurs ~20% of the time) down to just 19% would cut carbon emissions by ~100,000 Americans. Note: This is back of napkin math, done by an ex-rocket Scientist though, so we’ll take it. 
+
+In this first installment, you’ll hear from a Researcher (who’s using ground penetrating radar to detect buried landmines), an ex-Rocket Scientist, a Dzongkha fluent amateur gamer (Kuzu = Hello!), an ex-van living Scientist, a high-performance Data Science team coach who’s still very hands-on, and a data practitioner who values relationships, family, dogs, and pizza. —All of whom are currently Directors of Machine Learning with rich field insights.
```
Comment (osanseviero) [inline ml-director-insights.md:40]:
```suggestion
In this first installment, you’ll hear from a researcher (who’s using ground penetrating radar to detect buried landmines), an ex-Rocket Scientist, a Dzongkha fluent amateur gamer (Kuzu = Hello!), an ex-van living Scientist, a high-performance Data Science team coach who’s still very hands-on, and a data practitioner who values relationships, family, dogs, and pizza. —All of whom are currently Directors of Machine Learning with rich field insights.
```

**Review by osanseviero — APPROVED** [review]
I just did a light reading (did not read all in detail), but overall looks good.

As for the old commit, I think it should be ok once we do merge and squash.

**Pull request merged**